Sources/XCHammer/XcodeTarget.swift

```diff
@@ -469,7 +470,18 @@ public class XcodeTarget: Hashable, Equatable {
                         stopAfterNeedsRecursive ))
                 .filter { !$0.isEntitlementsDep }
 
-            return Array(Set(deps.flatMap { $0.myResources }))
+            // Install compiled bundles from BUILT_PRODUCTS_DIR
+            let bundleDeps: [XcodeTarget] = ([self] +
+                    self.transitiveTargets(map:
+                        self.targetMap, predicate: keepGoing ))
```

Comment:
Is there a way to skip transitive extraction of frameworks / visiting them? At the time of writing, a framework shoudn't pack resources for transitive deps that exist in other frameworks. Also hard to tell w/o a profiler but I suspected this and similar code paths might have N^2 looping

Reply:
There's probably a way to add a less greedy predicate here, will tackle that as follow up